2021-11-18 22:13:51,728 INFO    MainThread:9804 [wandb_setup.py:_flush():71] setting env: {}
2021-11-18 22:13:51,728 INFO    MainThread:9804 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-18 22:13:51,728 INFO    MainThread:9804 [wandb_init.py:_log_setup():357] Logging user logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_221351-3iu11mc7/logs/debug.log
2021-11-18 22:13:51,728 INFO    MainThread:9804 [wandb_init.py:_log_setup():358] Logging internal logs to /home/nicolas/workspace_josue/Pytorch-UNet/wandb/run-20211118_221351-3iu11mc7/logs/debug-internal.log
2021-11-18 22:13:51,729 INFO    MainThread:9804 [wandb_init.py:init():390] calling init triggers
2021-11-18 22:13:51,729 INFO    MainThread:9804 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-18 22:13:51,730 INFO    MainThread:9804 [wandb_init.py:init():435] starting backend
2021-11-18 22:13:51,730 INFO    MainThread:9804 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-18 22:13:51,776 INFO    MainThread:9804 [backend.py:ensure_launched():199] starting backend process...
2021-11-18 22:13:51,797 INFO    MainThread:9804 [backend.py:ensure_launched():205] started backend process with pid: 9824
2021-11-18 22:13:51,798 INFO    MainThread:9804 [wandb_init.py:init():444] backend started and connected
2021-11-18 22:13:51,799 INFO    MainThread:9804 [wandb_init.py:init():503] updated telemetry
2021-11-18 22:13:51,851 INFO    MainThread:9804 [wandb_init.py:init():533] communicating current version
2021-11-18 22:13:52,156 INFO    MainThread:9804 [wandb_init.py:init():538] got version response 
2021-11-18 22:13:52,157 INFO    MainThread:9804 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-18 22:13:52,506 INFO    MainThread:9804 [wandb_init.py:init():576] starting run threads in backend
2021-11-18 22:13:53,442 INFO    MainThread:9804 [wandb_run.py:_console_start():1693] atexit reg
2021-11-18 22:13:53,443 INFO    MainThread:9804 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-18 22:13:53,443 INFO    MainThread:9804 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-18 22:13:53,444 INFO    MainThread:9804 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-18 22:13:53,444 INFO    MainThread:9804 [wandb_init.py:init():603] run started, returning control to user process
2021-11-18 22:13:53,445 INFO    MainThread:9804 [wandb_run.py:_config_callback():911] config_cb None None {'epochs': 5, 'batch_size': 5, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-11-18 22:13:53,761 INFO    MainThread:9804 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-18 22:13:53,762 INFO    MainThread:9804 [wandb_run.py:_restore():1635] restore
2021-11-18 22:13:56,482 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 810
}

2021-11-18 22:13:56,719 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 810
}

2021-11-18 22:13:57,518 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 2906
}

2021-11-18 22:13:57,621 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 10317
}

2021-11-18 22:13:57,736 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 810
  total_bytes: 10317
}

2021-11-18 22:13:57,839 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}

2021-11-18 22:13:57,942 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}

2021-11-18 22:13:58,044 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}

2021-11-18 22:13:58,147 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}

2021-11-18 22:13:58,250 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}

2021-11-18 22:13:58,455 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}

2021-11-18 22:13:58,848 INFO    MainThread:9804 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10317
  total_bytes: 10317
}
local_info {
}

2021-11-18 22:13:59,906 INFO    MainThread:9804 [wandb_run.py:_append_files():2061] logging synced files
